XIPs/xip-21-transaction-reference-content-type.md

```diff
@@ -62,7 +58,7 @@ type TransactionReference = {
 
 ## Rationale
 
-The `networkId` provides details of the network used for the transaction, while the `reference` field contains the hash of the transaction on the network. These two fields should be enough to display a basic reference to the transaction. An optional `namespace` field can be used for a more human-readable description of the network.
+The `chainId` provides details of the network used for the transaction in [CAIP-2 format](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-2.md), while the `transactionHash` field contains the hash of the transaction on the network. These two fields should be enough to display a basic reference to the transaction. An optional `namespace` field can be used for a more human-readable description of the network.
```

Comment:
> An optional `namespace` field can be used for a more human-readable description of the network.

I believe the sentence is no longer necessary.

Reply:
Captured here for discussion upon Ry's return: https://community.xmtp.org/t/xip-21-on-chain-transaction-reference-content-type/532/10